modules/compute_instance/README.md

```diff
@@ -31,4 +31,4 @@ See the [simple](examples/compute_instance/simple) for a usage example.
 | available\_zones | List of available zones in region |
 | instances\_self\_links | List of self-links for compute instances |
 
-[^]: (autogen_docs_end)
\ No newline at end of file
+[^]: (autogen_docs_end)
```

Comment:
this is the old style doc generator marker, please replace it across the code base using [this function](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/concourse/build/developer-tools/build/scripts/task_helper_functions.sh#L186)

Reply:
Fixed